#m210:
# Publish all joint states -----------------------------------
joint_state:
  type: joint_state_controller/JointStateController
  publish_rate: 50

# Position Controllers ---------------------------------------

gimbal_yaw:
  type: effort_controllers/JointPositionController
  joint: /m210/gimbal_joint_yaw
  pid: {p: 0.1, i: 0.007, d: 0.000}

gimbal_roll:
  type: effort_controllers/JointPositionController
  joint: /m210/gimbal_joint_roll
  pid: {p: 0.1, i: 0.007, d: 0.000}

gimbal_pitch:
  type: effort_controllers/JointPositionController
  joint: /m210/gimbal_joint_pitch
  pid: {p: 0.1, i: 0.0001, d: 0.000}
